### DISCUSSION TITLE
Customize Directory Color

### DISCUSSION THREAD

**Discussion by patrick-5546** [lsd-rs/lsd#898] [Q&A] [answered]
Hello, when I use lsd directories are dark blue. Which key in the colors.yaml does this correspond to? Or is there a way to change the theme to one that has more contrast with black terminals?
![image](https://github.com/lsd-rs/lsd/assets/65756895/00493bcb-101b-4cd9-be2d-c0889a4432cf)

Also how do the values that are numbers work? Are they hex?

**Comment by zwpaper** [comment, marked as answer]
hi, the color format is explain in [readme](https://github.com/lsd-rs/lsd?tab=readme-ov-file#color-theme-file-content).

the file name color is specified by lscolors, we have not yet changed it to use the color theme.

I have this plan, have not yet have time to make it come true

**Reply by zzhaolei** [reply]
期待
